internal/embedded/embedded.go

```diff
@@ -7,7 +7,19 @@ import (
 //go:embed favicon.ico
 var favicon []byte
 
+//go:embed normalize.css
+var normalize string
+
 // Favicon returns the favicon data
 func Favicon() []byte {
 	return favicon
 }
+
+// DefaultCss returns the default css data
+func DefaultCss(css string) string {
```

Comment:
I think we can just use function names like `func Normalize() []byte` or `func Reset() []byte` since it's pretty clear where the context comes from.